main.py

```diff
@@ -27,101 +24,338 @@ async def play(ctx):
+    help="Bot informuje o obecnie granym utworze."
+)
+@commands.cooldown(1, 60, commands.BucketType.user)
+async def cjg(ctx):
+    embed = discord.Embed(title="Obecnie gramy:")
+    resp = requests.get("https://radio17.pl/api/stream/stats/extended")
+    json_resp = resp.json()
+    is_played = json_resp["current"]["artist"] + " - " + json_resp["current"]["title"]
+    embed.add_field(name=is_played, value="https://radio17.pl")
+    await ctx.channel.send(embed=embed)
+
+
+@slash.slash(
+    name="cjg",
+    description="Informuje co obecnie jest grane w radiu."
+)
 async def _cjg(ctx: SlashContext):
     embed = discord.Embed(title="Obecnie gramy:")
     resp = requests.get("https://radio17.pl/api/stream/stats/extended")
-    grane = " " + resp.json()['current']['artist'] + " - " + resp.json()['current']['title']
-    embed.add_field(name=grane, value="https://radio17.pl")
+    json_resp = resp.json()
+    is_played = json_resp["current"]["artist"] + " - " + json_resp["current"]["title"]
+    embed.add_field(name=is_played, value="https://radio17.pl")
     await ctx.send(embed=embed)
 
 
-@slash.slash(name="cbg",
-             description="Informuje co będzie grane za chwilę w radiu.")
-async def _cbg(ctx: SlashContext):
-    embed = discord.Embed(title="Za chwilę będziemy grać:")
-    resp = requests.get("https://radio17.pl/api/stream/stats/extended")
-    if resp.json()['next'] is not None:
-        bedzie = " " + resp.json()['next']['artist'] + " - " + resp.json()['next']['title']
-        embed.add_field(name=bedzie, value="https://radio17.pl")
-        await ctx.send(embed=embed)
+@bot.command(
+    brief="Wysyła wiadomość na czat strony Radia1.7."
+)
+@commands.cooldown(1, 60, commands.BucketType.user)
+async def send(ctx, part):
+    command, author, message = ctx.message.content.rpartition(part)
+    message = message[1:]
+    msg = {"author": author, "content": message}
+    resp = requests.post("https://radio17.pl/api/chat/message", json=msg)
+    if resp.status_code == 201:
+        await ctx.channel.send("Twoja wiadomość o treści: \"%s\" została wysłana." % message)
     else:
-        await ctx.send("No nie wiem co będzie grane.")
+        await ctx.channel.send("Niestety wiadomość nie mogła zostać wysłana, spróbuj ponownie później")
 
 
-@slash.slash(name="play", description="Bot łączy się z kanałem głosowym.")
-async def _play(ctx):
-    try:
-        connected = ctx.author.voice
-        if not connected:
-            await ctx.send("Wbijaj na głosowy, by słuchać radia!")
-            return
-        channel = ctx.author.voice.channel
+@slash.slash(
+    name="send",
+    description="Wysyła wiadomość na czat strony Radia1.7"
+)
+async def _send(ctx: SlashContext, author, message):
+    msg = {"author": author, "content": message}
+    resp = requests.post("https://radio17.pl/api/chat/message", json=msg)
+    if resp.status_code == 201:
+        await ctx.send("Twoja wiadomość o treści: \"%s\" została wysłana." % message)
```

Comment:
```suggestion
        await ctx.send(f'Twoja wiadomość o treści: "{message}" została wysłana.')
```
Temu ja zazwyczaj stringi daję w apostrofach, bo częściej się zdarzy cudzysłów w stringu niż apostrof.

Ale to nie o tym - jak już interpolujesz w P3, to raczej f-stringiem, szczególnie jak targetujesz pythona 3.8

Reply:
Wyżej też takie jedno było